### PR TITLE
Feature/survey personal frag

### LINKED ISSUE #29: Reset to family tab upon switching screen orientation
When switching orientation, tab automatically gets set back to Family Tab

To implement:
Create a view model for tab bar and save the state before destroying the activity

### PR DESCRIPTION
**Pull request by bhylak**
Fixed a lot of problems with the implementation of our tab switching. For example, all tabs are referenced by class, and then looked up in the fragment manager by the class name.

Added survey question creation based off of the survey schema defined in the server. Added a progress bar to track the progress of the background questions. (Should also work for indicators, but it's hard to say for sure until it is tested)

We wanted to use a recycler view (specifically, a [DiscreteScrollView](https://github.com/yarolegovich/DiscreteScrollView)) for the background questions to fit the UI specs we developed a bit better, but we decided to wait until making some fixes to our model (combining economic and personal questions) because it's too hard to wrangle separate lists in the recycler view, and it's a problem we were planning on resolving soon anyway.

Because of that, we didn't pay too much attention to styling in the background questions but it is functional.

Also fixes #29

### CODE REVIEW

File: app/src/main/java/org/fundacionparaguaya/advisorapp/fragments/StackedFrag.java [hunk 1]
```diff
@@ -44,7 +44,7 @@ public void onAttach(Context context)
         }
         catch (ClassCastException e)
         {
-            throw new ClassCastException("Parent activity or fragment must implement NavigationListener");
+           // throw new ClassCastException("Parent activity or fragment must implement NavigationListener");
```
Comment (krconv) [inline app/src/main/java/org/fundacionparaguaya/advisorapp/fragments/StackedFrag.java]: Any reason for commenting this out? Or was it for debuging?

File: app/src/main/java/org/fundacionparaguaya/advisorapp/viewmodels/SharedSurveyViewModel.java [hunk 2]
```diff
@@ -45,7 +45,7 @@ public SharedSurveyViewModel(SurveyRepository surveyRepository, FamilyRepository
         mSurveyRepository = surveyRepository;
         mFamilyRepository = familyRepository;
 
-        mProgress.setValue(new SurveyProgress());
+        //Progress.setValue(new SurveyProgress(0, ""));
```
Comment (krconv) [inline app/src/main/java/org/fundacionparaguaya/advisorapp/viewmodels/SharedSurveyViewModel.java]: Any reason for commenting this one out?

File: app/src/main/java/org/fundacionparaguaya/advisorapp/fragments/StackedFrag.java (hunk 1 above)
Comment (bhylak) [reply app/src/main/java/org/fundacionparaguaya/advisorapp/fragments/StackedFrag.java]: Good catch, didn't mean to leave that commented.

File: app/src/main/java/org/fundacionparaguaya/advisorapp/viewmodels/SharedSurveyViewModel.java (hunk 2 above)
Comment (bhylak) [reply app/src/main/java/org/fundacionparaguaya/advisorapp/viewmodels/SharedSurveyViewModel.java]: Meant to delete this line.

**Pull request merged**